Cesium.CodeGen/Ir/Expressions/Constants/CharConstant.cs

```diff
@@ -41,10 +41,11 @@ private static char UnescapeCharacter(string text)
             'n' => '\n',
             'r' => '\r',
             't' => '\t',
-            'v' => '\v',
+            '?' => '?',
+            'v' => (char)0x0b,
             'x' => (char)int.Parse(text.AsSpan(2), System.Globalization.NumberStyles.AllowHexSpecifier),
             > '0' and < '9' => (char)Convert.ToInt32(text.Substring(2), 8),
-            _ => throw new CompilationException($"Unknown escape sequence '{text}'"),
+            _ => text[1],
```

Comment:
Let's warn on it at least? This is not valid according to the standard, even though some compilers indeed get it passed with a warning.

Reply:
Ok I dig into this for a bit, and decided to open a new issue on it. Anyway, this deserves some more polishing, let's do it separately.

Reply:
See #699.